style.css

```diff
@@ -436,7 +449,7 @@ padding: auto;
     display: flex;
     flex-direction: column;
     gap: 20px;
-    margin: 0px -90px 0px 0px;
+    /* margin: 0px -90px 0px 0px; */
```

Comment:
The design of the site has a lot of elements shifted off to the left/right sides and I noticed we're using margins or padding in a lot of places to achieve this.

Generally, I would recommend putting those elements inside a `<div>` that is `relative`-positioned. Then the element we want to shift can be `absolute`-positioned relative to its parent.

Reply:
Yeah, following the draft mark-ups from Figma, I had a hard time getting the elements in the right position. Totally forgot about "relative".